py/dml/provisional.py

```diff
@@ -135,6 +135,34 @@ class simics_util_vect(ProvisionalFeature):
     stable = True
     dml12 = True
 
+
+@feature
+class explicit_object_extensions(ProvisionalFeature):
+    '''<a id="explicit_object_extensions"/>
+    This feature extends the DML syntax for object declarations to distinguish
```

Comment:
newline after `<a`, otherwise markdown is messed up in the first paragraph.

Reply:
wait-- you mean
```
<a
 id="explicit_object_extensions"/>
```
? That's messed up, man...

Reply:
... in fact I can't see a `<a`-newline pattern being used anywhere?